src/lib/mongoBackend/mongoRegistrationGet.cpp

```diff
@@ -176,11 +177,10 @@ static void setAttributes(ngsiv2::Registration* regP, const mongo::BSONObj& cr0)
 * If we have more than one, then the Registration is made in API V1 as this is not
 * possible in V2 and we cannot respond to the request using the current implementation of V2.
 * This function will be changed to work in a different way once issue #3044 is dealt with.
-*
 */
-static bool setDataProvided(ngsiv2::Registration* regP, const mongo::BSONObj& r, bool arrayAllowed)
+static bool setDataProvided(ngsiv2::Registration* regP, const mongo::BSONObj& r, bool arrayAllowed, const char* dbLabel)
```

Comment:
Function is overparametriced, `dbLabel` is not needed at the field used for context registrations is not a parameter but fixed by the database model.

Reply:
Fixed in 5d851ef